exercises/bank-account/examples/success-standard/src/BankAccount.hs

```diff
@@ -3,18 +3,18 @@ module BankAccount ( BankAccount
                    , getBalance, incrementBalance) where
 import Control.Concurrent.STM (TVar, atomically, newTVar, readTVar, readTVarIO, writeTVar)
 
-newtype BankAccount = BankAccount { unBankAccount :: TVar (Maybe Int) }
+newtype BankAccount = BankAccount { unBankAccount :: TVar (Maybe Integer) }
 
 openAccount :: IO BankAccount
 openAccount = atomically $ BankAccount <$> newTVar (Just 0)
 
 closeAccount :: BankAccount -> IO ()
 closeAccount = atomically . flip writeTVar Nothing . unBankAccount
 
-getBalance :: BankAccount -> IO (Maybe Int)
+getBalance :: BankAccount -> IO (Maybe Integer)
 getBalance = readTVarIO . unBankAccount
 
-incrementBalance :: BankAccount -> Int -> IO (Maybe Int)
+incrementBalance :: BankAccount -> Integer -> IO (Maybe Integer)
```

Comment:
The PR/commit do not mention why `Int` was changed into `Integer`.

Investigating, it seems that the stub has used `Integer` ever since it was introduced in 0bfdca039de71a02f6352df14ef9175e44f4df51 (2016-07-13), but the example solution of d956febd4043e1091300b1c3440a1fbd28dd52c1 (2013-10-25) uses `Int`.

So on the premise of consistency alone, this is a reasonable change.

Reply:
Ah, yes, sorry! Yup, I was just fixing the inconsistency between stub and example.